javascript/ql/examples/queries/dataflow/TemplateInjection/TemplateInjection.ql

```diff
@@ -14,7 +14,7 @@ import DataFlow::PathGraph
 /**
  * Gets the name of an unescaped placeholder in a lodash template.
  *
- * For example, the string `<h1><%= title %></h1>` contains the placeholder `title`.
+ * For example, the string "<h1><%= title %></h1>" contains the placeholder "title".
```

Comment:
I assume the `<h1>...` bit was originally in backticks so as to preserve the formatting, and now it's not. Perhaps it should be changed to
```
`"<h1><%= title %></h1>"`
```
so that both string-ness and pre-formatting are evident.